Tests/SwerverTests/ActionSpec/HeadActionSpec.swift

```diff
@@ -9,8 +9,8 @@ class HeadActionSpec: QuickSpec {
             let responseBuilder = ResponseBuilder()
             let headAction = HeadAction()
 
-            it("can dispatch to response builder to build a response for a successful HEAD request") {
-                let expectResponse = headAction.dispatch()
+            it("can execute to response builder to build a response for a successful HEAD request") {
```

Comment:
What does "it can execute to response builder" mean?

Reply:
i meant to replace all the dispatch() with the name execute(). will fix that.

Reply:
fixed